skorch/tests/test_classifier.py

```diff
@@ -266,6 +266,19 @@ def test_not_fitted_raises(self, net_cls, module_cls, data, method):
                "before using this method.")
         assert exc.value.args[0] == msg
 
+    def test_not_fitted_raises(self, net_cls, module_cls):
+        from sklearn.utils.validation import check_is_fitted
+        from sklearn.exceptions import NotFittedError
+    
+        net = net_cls(module_cls)
+        with pytest.raises(NotFittedError) as exc:
+            check_is_fitted(net)
+
+        msg = ("This NeuralNetBinaryClassifier instance is not fitted yet. "
+                "Call 'fit' with appropriate arguments before "
+                "using this estimator.")
```

Comment:
The indentation is a bit off in the 2nd and 3rd line. Overall, I think it's easier and more consistent to use hanging indents here and below:

```suggestion
        msg = (
            "This NeuralNetBinaryClassifier instance is not fitted yet. "
            "Call 'fit' with appropriate arguments before "
            "using this estimator."
        )
```